libraries/AC_Autorotation/AC_Autorotation.cpp

```diff
@@ -380,3 +437,89 @@ float AC_Autorotation::calc_speed_forward(void)
     return speed_forward;
 }
 
+void AC_Autorotation::flare_controller(float dt)
+{
+		
+  // Specify forward velocity component and determine delta velocity with respect to time
+    _speed_forward = calc_speed_forward(); //(cm/s)
+    _delta_speed_fwd = _speed_forward - _speed_forward_last; //(cm/s)
+    _speed_forward_last = _speed_forward; //(cm/s)
+    float current_alt = _inav.get_position_z_up_cm();
+    _desired_speed = linear_interpolate(0.0f, _flare_entry_speed, current_alt, 0.0f, _param_flr_alt);
+
+	// get p
+	_vel_p = _p_fw_vel.get_p(_desired_speed - _speed_forward);
+
+    //calculate acceleration target based on PI controller
+    _accel_target = _vel_p ;
+
+    // filter correction acceleration
+    _accel_target_filter.set_cutoff_frequency(10.0f);
```

Comment:
Don't think we should be hard coding filters.  Depends very much on the users setup as to what cutt offs should be set.